etc/kayobe/environments/ci-multinode/seed.yml

```
@@ -26,6 +24,6 @@ snat_rules_default:
     source_ip: "{{ ansible_facts.default_ipv4.address }}"
 snat_rules_manila:
   - interface: "{{ storage_interface }}"
-    source_ip: "{{ ansible_facts[storage_interface].ipv4.address }}"
+    source_ip: "{{ ansible_facts[storage_interface].ipv4.address if ansible_facts[storage_interface].ipv4.address is defined else none }}"
```

Can you just do this?
```suggestion
    source_ip: "{{ ansible_facts[storage_interface].ipv4.address | default }}"
```

---

I gave it a go and it looks good